core/sys/posix/netinet_in.odin

```diff
@@ -45,20 +45,18 @@ when ODIN_OS == .Darwin || ODIN_OS == .FreeBSD || ODIN_OS == .NetBSD || ODIN_OS
 	}
 
 	when ODIN_OS == .Linux {
+
 		sockaddr_in :: struct {
 			sin_family: sa_family_t, /* [PSX] AF_INET (but a smaller size) */
 			sin_port:   in_port_t,   /* [PSX] port number */
 			sin_addr:   in_addr,     /* [PSX] IP address */
-			sin_zero: [size_of(sockaddr) -
-				u16 -
-				size_of(in_port_t) -
-				size_of(in_addr)]c.char,
+			sin_zero:   [8]c.char,
 		}
```

Comment:
IIRC (from when I did it) that whole expression comes out as `8` so they should be equivalent.

Reply:
I thought having that be a 1:1 formula conversion from C would make it automatically compatible in case this ever changes in glibc. But I don't know how often, if ever, that changes due to backwards compatibility.
So I'll mark this as resolved.